spec/paypal_express/remote/baid_helpers.rb

```diff
@@ -0,0 +1,50 @@
+module BaidHelpers
+  def baid_setup(payment_processor_account_id = nil)
+    @call_context = build_call_context
+
+    options = {}
+    unless payment_processor_account_id.nil?
```

Comment:
You could simplify and remove this check, since `options[:payment_processor_account_id]` would return `nil` anyways.